<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Veyon Community</author>
 <class>QuickPollsDialog</class>
 <widget class="QDialog" name="QuickPollsDialog">
  <property name="windowTitle">
   <string>Create Quick Poll</string>
  </property>
  <property name="windowIcon">
   <iconset resource="quickpolls.qrc">
    <normaloff>:/quickpolls/poll.png</normaloff>:/quickpolls/poll.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Enter your question and the possible answers below.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="questionLabel">
       <property name="text">
        <string>Question:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="questionLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="answersLabel">
     <property name="text">
      <string>Answers:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="answersListWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addAnswerButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeAnswerButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="quickpolls.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QuickPollsDialog</receiver>
   <slot>accept()</slot>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QuickPollsDialog</receiver>
   <slot>reject()</slot>
  </connection>
 </connections>
</ui>